src/components/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,27 @@
+import Grid from '@mui/material/Grid';
+
+import BottomBar from './BottomBar';
+import ListOfHouses from './ListOfHouses';
+import MapHouses from './MapSide';
+import TopBar from './TopBar';
+
+function HomePage() {
+  return (
+    <Grid container>
```

Comment:
The layout becomes a little off on screens wider than 2380px. You could consider locking the content between `TopBar` and `BottomBar` so that it doesn't spread more on wider screens.